docs/en/ingest-management/commands.asciidoc

```diff
@@ -834,6 +835,63 @@ elastic-agent install --url=https://fleet-server:8220 \
   --certificate-authorities=/path/to/ca.crt
 ----
 
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-otel-command]]
+== elastic-agent otel
+
+preview::[]
+
+Run {agent} as an <<otel-agent,OpenTelemetry Collector>>. 
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent otel [flags]
+elastic-agent otel [command]
```

Comment:
we may want to include a note that `./otelcol` is also available as it calls `elastic-agent otel` and passes any args